### ISSUE TITLE
Tabs - Missing aria-selected="false"

### ISSUE THREAD

**Issue by greenc05** [bbc/gef-docs#89]
The enhanced mark up (https://bbc.github.io/gef-docs/components/tabs/#enhanced-markup) is missing aria-selected="false" on each of the unselected tabs (as per the W3C spec, 'The active tab element has the state aria-selected set to true and all other tab elements have it set to false.'). This is also missing from the reference implementation (https://bbc.github.io/gef-docs/components/demos/tabs/).

Without this it causes a bug with Jaws and NVDA where when navigating via the tab key through the tabs, it announces that the tabs are all selected, this is confusing as it announced the same if the tab is active.

**Comment by greenc05** [comment]
Any chance you could also look at this please @simonsinclair?

**Comment by simonsinclair** [comment]
Absolutely, @greenc05.

**Comment by micmath** [comment]
Fixed
